### DISCUSSION TITLE
Welcome to dict-pt-br Discussions!

### DISCUSSION THREAD

**Discussion by dehilsterlexis** [VisualText/dict-pt-br#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dehilsterlexis** [comment]
@ribasnath @ebritto25 voces podem postar perguntas aqui quando voces tem perguntas.

**Comment by dehilsterlexis** [comment]
@ribasnath @ebritto25 por favor atualize o repositorio. Fiz algumas mudancas. Obrigado!

**Comment by dehilsterlexis** [comment]
@ribasnath @ebritto25 por favor atualize o repositorio mais uma vez. Fiz algumas mudancas. Obrigado!

**Comment by dehilsterlexis** [comment]
@ribasnath @ebritto25 por favor atualize o repositorio mais uma vez. Fiz algumas mudanças. Eu vou marcar uma reunião.

**Comment by ddehilster** [comment]
@ribasnath @ebritto25 quando é uma boa hora para vocês se encontrarem?

**Reply by ebritto25** [reply]
Qualquer dia (menos às quarta-feiras) após as 18hrs BRT é ótimo para mim. Antes disso eu teria que ver se não tenho alguma outra reunião.

**Reply by ribasnath** [reply]
Excluindo hoje (quinta-feira) que estarei ocupada o dia todo, também posso qualquer dia após às 17h, outros horários teria que ver a disponibilidade previamente.

**Comment by dehilsterlexis** [comment]
@ribasnath @ebritto25 Here is the recording of our last session: https://youtu.be/mqCY69CxOEI

**Comment by ebritto25** [comment]
Hey @ddehilster ,

Eu e a @ribasnath conversamos e ela irá padronizar as abreviações e eu vou extrair as outras formas dos verbos irregulares.

**Comment by dehilsterlexis** [comment]
Super!

**Comment by ddehilster** [comment]
@ribasnath @ebritto25 a ultima versao do analizdor esta funcionando. Verifiquei agora. A unica palavra que nao esta saindo e' dormir. Faca um clone limpa para gaurantir que voces tem a ultima versao.

**Comment by ddehilster** [comment]
@ribasnath @ebritto25 por favor faz um "fork" do repositorio do dicionario portugues. Nathalia, por favor adiciona a sua tabela para o README to nivel top. Pode fazer as modificacoes no fork e push para o repositorio. Se precisar ajuda em fazer isso, me fala. Obrigado!

**Reply by ribasnath** [reply]
feito!

**Comment by dehilsterlexis** [comment]
@ribasnath lembrando para eliminar os logs e debug.txt do PR.

**Reply by ribasnath** [reply]
já foram removidos

**Comment by dehilsterlexis** [comment]
@ribasnath tem 3 arquivos que tem que ser removidos ainda: 
![image](https://github.com/VisualText/dict-pt-br/assets/10133951/2782ab2e-71f6-48f7-9279-1baaf02f282f)

**Comment by dehilsterlexis** [comment]
@ribasnath @ebritto25 vamos reunir sexta feira dia 2 de fev falar sobre os próximos passos? 18:30 esta bem para voces?

**Reply by ebritto25** [reply]
Pode ser às 18:30 por mim

**Comment by dehilsterlexis** [comment]
@ribasnath @ebritto25 tudo nem? Como está andando as suas tarefas?

**Comment by dehilsterlexis** [comment]
@ribasnath @ebritto25 tudo nem? Vamos reunir logo?

**Comment by ebritto25** [comment]
Hey David

Podemos fazer uma reunião qualquer dia dessa semana, com exceção de
quarta-feira.

Em dom., 3 de mar. de 2024 13:25, David de Hilster ***@***.***>
escreveu:

**Comment by ribasnath** [comment]
Oi David,

Estou disponível todos os dias desta semana.

**Comment by ddehilster** [comment]
@ebritto25 @ribasnath boa tarde! marquei uma reunião quarta-feira as 6:30 da tarde. Vocês deviam ter recebidos um email. Esta ok com voces?

**Comment by ebritto25** [comment]
Como eu falei no outro email, infelizmente Quarta-feira não funciona pra
mim. Mas se não der pra remarcar eu posso ver a gravação depois, sem
problemas.

Em ter., 5 de mar. de 2024 12:57, David de Hilster ***@***.***>
escreveu:

**Comment by ddehilster** [comment]
@ebritto25 @ribasnath carumba! Eu esqueci! Mudei para quinta.

**Comment by ebritto25** [comment]
David, acho que você não mandou o convite para a nova reunião, só cancelou
a anterior

Em ter., 5 de mar. de 2024 13:23, David de Hilster ***@***.***>
escreveu:

**Reply by dehilsterlexis** [reply]
Ok. Eu vou dar um olhado.

**Reply by dehilsterlexis** [reply]
@ebritto25 obrigado! Você tinha razão!

**Comment by ddehilster** [comment]
@ebritto25 @ribasnath boa tarde! Voltei das férias. Vamos reunir a semana que vem sobre os próximos passos con dicionário?

**Comment by fallsdevil** [comment]
hey, preciso de uma ajuda muito específica. eu poso usar esse arquivo zip como um arquivo .dict? acontece que uso um teclado chamado heliboard que permite importar dicionário do usuário, mas não estou satisfeito com as sugestões de palavras, e mesmo importando um que eles recomendam, não adianta, está desatualizado e não contém alguns verbos básicos conjugados, como os terminados em "ei" (pensei, joguei, parei, etc) e eu queria encontrar algum arquivo que tenha palavras contraídas de verbo + pronome oblíquo "vê-lo, conhecê-la, concluí-las" pra funcionar  semelhante as sugestões e correções do teclado nativo da Samsung, mas até o dicionário do libreOffice nao tinha esse formato de verbos...
